network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/formassembly_body.html

```diff
@@ -46,6 +50,8 @@
             wFORMS.behaviors.condition.isConditionalSubmitAllowed = true;
         </script></div>
         <form method="post" action="https://mozillafoundation.tfaforms.net/api_v2/workflow/processor" class=" labelsAbove" id="9" role="form">
+            <label class="label tw-body-small tw-italic tw-mt-2">{% blocktrans %} * indicates a required field {% endblocktrans %}</label>
```

Comment:
Hi @danielfmiranda after looking at this PR closely, I'm thinking it might be better to move this line outside of this file (i.e., move to `petition.html` before the `{% include ...` on line 28) because

1. This line is a completely new element we are adding to the page and not something came with the code snippets from FormAssembly
2. Including this line here means every time we update the FA form we'll have to re-add the `* indicates a required field` note in this file which creates more code maintenance overhead.

I would also suggest that we switch to use `<p>` instead of `<label>` (we probably need to adjust the vertical margin a bit as well)

Reply:
Thanks for the feedback Mavis! Moving it outside of the template makes sense to me so I have done that in the latest commit. 

I also updated the `<label>` to a `<p>`, here is a screenshot confirming that visually nothing has changed except the vertical spacing of the element (made a small change from `tw-mt-2` to `tw-my-4`.)

![Screenshot 2023-07-17 at 16-19-35 Tell Slack Make DMs Safe](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/5582cd9e-1abf-4ec4-a205-875ed655e5ce)